SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -775,9 +775,17 @@ void USpatialNetDriver::CleanUpServerConnectionForPC(APlayerController* PC)
 		   TEXT("While trying to clean up a PlayerController, its client connection was not found and thus cleanup was not performed"));
 }
 
-void USpatialNetDriver::OnActorSpawned(AActor* Actor) const
+void USpatialNetDriver::OnActorSpawned(AActor* Actor)
```

Comment:
Just wondering why the loss of "const". Does PackageMap->TryResolveObjectAsEntity somehow modify something on the SpatialNetDriver?

Reply:
Ah no, the loss of const was because a previous revision did modify the netdriver, I'll add it